tests/unit/test_repos_release.py

```diff
@@ -157,3 +177,7 @@ def test_edit_with_label(self):
         assert json.loads(kwargs['data']) == {
             'name': 'new name', 'label': 'label'
             }
+
+    def test_str(self):
```

Comment:
> Also let's not bother with this test. It's the same as a test for __repr__

@sigmavirus24 there's not an existing `__repr__` test for this class; should this test still be omitted ?